Dockerfile.client

```diff
@@ -24,10 +24,11 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-# Default setting is building on ubuntu:18.04
+# Default setting is building on nvidia/cuda:10.1-devel-ubuntu18.04
 ARG UBUNTU_VERSION=18.04
```

Comment:
Remove this arg? UBUNTU_VERSION no longer used?

Reply:
Added to make the user choose between CUDA Ubuntu 18.04 and CUDA Ubuntu 16.04 images. Removing and modifying documentation